index.js

```diff
@@ -0,0 +1,4 @@
+const userName = prompt('Cuál es tu nombre?') || 'Pepe'
+
+
+alert(`Hola ${userName}!` );
```

Comment:
En este caso me parece que sería mejor que no uses interpolación de strings.
```suggestion
alert('Hola ' + userName );
```